setup.cfg

```diff
@@ -20,6 +20,7 @@ description-file = README.rst
 
 [options]
 packages = find:
+include_package_data = True
```

Comment:
@oesteban Just FYI this was the issue. See https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords